### PR TITLE
[lldb] Fix TypeSystemSwiftTypeRe for NDEBUG builds

### PR DESCRIPTION
**Pull request by JDevlieghere**

### CODE REVIEW

**Comment by JDevlieghere** [comment]
@swift-ci please test

**Review by adrian-prantl — APPROVED** [review]
Wow. Could you please make sure to cherry-pick this to swift/release/5.4 if applicable?

File: lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp [hunk 1]
```diff
@@ -2154,11 +2155,11 @@ TypeSystemSwiftTypeRef::GetBitSize(opaque_compiler_type_t type,
     return {};
   };
   FALLBACK(GetBitSize, (ReconstructType(type), exe_scope));
-  if (exe_scope && exe_scope->CalculateProcess())
+  if (exe_scope && exe_scope->CalculateProcess()) {
```
Comment (adrian-prantl) [inline lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp:2158]: This one is NFC, right? Still a good idea, of course.

Comment (JDevlieghere) [reply lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.cpp:2158]: Probably, I wasn't sure if the whole `VALIDATE_AND_RETURN` would get compiled out or not, if not then this is indeed a NO-OP.

**Comment by JDevlieghere** [comment]
> Could you please make sure to cherry-pick this to swift/release/5.4 if applicable?

The 5.4 branch is fine. I remember fixing something similar last rebranch and I probably just forgot to cherrypick it to `swift/next`.

**Pull request merged**